thirdparty/sdcv/CMakeLists.txt

```diff
@@ -29,16 +32,10 @@ if(DEFINED ENV{ANDROID})
     set(PATCH_CMD sh -c "${PATCH_CMD}")
 endif()
 
-if(NOT DEFINED ENV{ANDROID} AND NOT DEFINED ENV{DARWIN})
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -lpthread -lrt")
-endif()
-
 # because cmake needs all kinds of annoying special cmake variables
 set(CMAKE_EXE_LINKER_FLAGS "${LDFLAGS} -static-libstdc++")
 if(DEFINED ENV{DARWIN})
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -framework CoreFoundation -framework Security")
-else()
-    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -static-libgcc")
```

Comment:
Looks like it originates with this commit: https://github.com/koreader/koreader-base/commit/db68822f9f1fd2472701b8d0c0677fe531b56333

That might mean it should be added to that Android line with the `-static` flag below.